daemons/controld/controld_control.c

```diff
@@ -233,7 +233,7 @@ crmd_exit(crm_exit_t exit_code)
     controld_globals.fsa_message_queue = NULL;
 
     controld_free_node_pending_timers();
-    controld_election_fini();
+    election_reset(controld_cluster); // Stop any election timer
```

Comment:
Why here and not in `attrd`? I take your word for it so it's not important enough to figure out myself. I'm just curious and assuming you know off hand.

I guess because another election might trigger while we're draining the controller mainloop later in this function, while there's no such possibility in `attrd` shutdown?

Reply:
Yep, exactly